common/CommandControl.cpp

```diff
@@ -35,9 +35,7 @@ void LockManager::generateLockedCommandList()
     std::string command;
     for (std::size_t i = 0; i < commandList.size(); i++)
     {
-        // just an extra check to make sure any whitespace does not sniff in command
-        // or else command will not be recognized
-        command = Util::trim_whitespace(commandList[i]);
```

Comment:
Should trimming still happen? Why not use `Util::trimmed()` ?

Reply:
I checked and `tokenize` does not include any space so its already trimmed no need to check again.